frontend/src/features/workspace/components/CreateFormModal/CreateFormModalContent/SaveSecretKeyScreen.tsx

```
@@ -55,7 +37,10 @@ const useSaveSecretKeyDefault = () => {
 
   const queryClient = useQueryClient()
   const navigate = useNavigate()
-  const formId = queryClient.getQueryData(workspaceKeys.lastCreatedForm)
+
+  const formTitle = useWatch({ control, name: 'title' })
+  const formId: string =
+    queryClient.getQueryData(workspaceKeys.lastCreatedForm) ?? ''
```

Seems like there could be a scenario where formId is `''`, which gets passed into `SaveSecretKeyContent` and used in the hook. I might have missed some context/behaviour here - should it be the case that `formId` is always a 24-char uuid and if it's not found we deal with it?

---

In this case, i think its fine. 


Context: 
This form id is used mainly for generating the file name of the secret key text file eg, `secretKey_<formId>_<formname>.txt` 

Tracing the previous code, the `const formId = queryClient.getQueryData(workspaceKeys.lastCreatedForm)` could have returned `undefined` which was not handled, potentially leading to `secretKey_<formId>_undfined.txt` being the file name. 

In this case, if the form name is not known, it would generate ``secretKey_<formId>_.txt` instead, which seems fine as well. 

Rationale: 
- It might be better to generate a secretKey txt file with no form name rather than explicitly block/throw an error when the form name is not found in our TanStack cache. 
- It is currently the same as existing behavior (out of scope for this PR) except that instead of `undefined` -> it just leaves the unknown form name as empty.